setup.py

```diff
@@ -63,5 +63,6 @@ def run(self):
 
 setup(
     distclass=Distribution,
+    setup_requires=['setuptools_scm'],
```

Comment:
This is the main thing that fixes dynamic versioning.

Reply:
why do you still require `setup.py` and not use just `pyproject.toml`?

Reply:
@carlosggarcia not sure why I can't respond to your last comment. We do use pyproject.toml, but this is the only way I have found to make sure that the dynamic version numbers are assigned.

Reply:
ok. weird. I think in TJPCov we have dynamic versioning and works fine with just `pyproject.toml`. I couldn't see any obvious difference that might cause this, though.

Reply:
Interesting. Well, if you can ever look into it, that'll be great. For now this is as far as I can get, and I think it is relatively urgent that we fix this! Our conda and pip releases are severely out of date.